package.json

```diff
@@ -27,27 +27,26 @@
     "lib": "lib"
   },
   "scripts": {
-    "prepublish": "npm run compile",
+    "update-version": "node -e \"console.log('module.exports = \\'' + require('./package.json').version + '\\';')\" > lib/version.js",
+    "version": "npm run update-version && git add lib/version.js && git commit -am \"Updating version\"",
     "publish": "git push origin && git push origin --tags",
```

Comment:
We had it in the docs but apparently it disappeared. There is so much noise in the main repo though that I don't think an automated changelog makes a lot of sense in this case.